### PR TITLE
dm: support ignore online-ddl in precheck

### PR DESCRIPTION
**Pull request by Ehco1996**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

as title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Ehco1996** [comment]
could you please take another look for this pr ? thanks 🧡🧡🧡
@sunzhaoyang

**Comment by Ehco1996** [comment]
@TomShawn ptal

**Comment by Ehco1996** [comment]
ping @ran-huang  ptal

File: dm/dm-precheck.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ tiup dmctl check-task ./task.yaml
 |table_schema|检查上游 MySQL 表结构的兼容性|
 |schema_of_shard_tables|检查上游 MySQL 多实例分库分表的表结构一致性|
 |auto_increment_ID|关闭检查上游 MySQL 多实例分库分表的自增主键冲突|
+|online_ddl|关闭检查上游是否处于 [Online-DDL](/dm/feature-online-ddl.md) 过程中|
```
Comment (ran-huang) [inline dm/dm-precheck.md:127]: “关闭”应该删掉吧。这个表里有的 row 没写“关闭”，有的写了“关闭”，但我理解它们都是一个意思。

Comment (Ehco1996) [inline dm/dm-precheck.md:127]:
```suggestion
|online_ddl|检查上游是否处于 [Online-DDL](/dm/feature-online-ddl.md) 过程中|
```

Comment (Ehco1996) [reply dm/dm-precheck.md]: fixed

File: dm/dm-precheck.md [hunk 2]
```diff
@@ -124,6 +124,7 @@ tiup dmctl check-task ./task.yaml
 |table_schema|检查上游 MySQL 表结构的兼容性|
 |schema_of_shard_tables|检查上游 MySQL 多实例分库分表的表结构一致性|
 |auto_increment_ID|关闭检查上游 MySQL 多实例分库分表的自增主键冲突|
```
Comment (ran-huang) [inline dm/dm-precheck.md:126]:
```suggestion
|auto_increment_ID|检查上游 MySQL 多实例分库分表的自增主键冲突|
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**